src/midi.ts

```diff
@@ -15,9 +15,10 @@ import * as jsmidgen from 'jsmidgen';
 export const midi = (
   notes: NoteObject[],
   fileName: string | null = 'music.mid',
-  bpm?: number
+  bpm?: number,
+  trackName?: string
 ): string | HTMLAnchorElement | undefined => {
-  const file = createFileFromNotes(notes, bpm);
+  const file = createFileFromNotes(notes, bpm, trackName);
```

Comment:
I have an alternative idea, how bout using the fileName (without the ".mid") here instead of a new var? This way we wont introduce yet another variable.